test/functional/test_runner.py

```diff
@@ -425,15 +426,14 @@ def get_next(self):
         while True:
             # Return first proc that finishes
             time.sleep(.5)
-            for j in self.jobs:
-                (name, time0, proc, testdir, log_out, log_err) = j
-                if os.getenv('TRAVIS') == 'true' and int(time.time() - time0) > 20 * 60:
-                    # In travis, timeout individual tests after 20 minutes (to stop tests hanging and not
-                    # providing useful output.
+            for job in self.jobs:
+                (name, start_time, proc, testdir, log_out, log_err) = job
+                if os.getenv('TRAVIS') == 'true' and int(time.time() - start_time) > TRAVIS_TIMEOUT_DURATION:
+                    # In travis, timeout individual tests (to stop tests hanging and not providing useful output).
                     proc.send_signal(signal.SIGINT)
                 if proc.poll() is not None:
                     log_out.seek(0), log_err.seek(0)
-                    [stdout, stderr] = [l.read().decode('utf-8') for l in (log_out, log_err)]
+                    [stdout, stderr] = [file.read().decode('utf-8') for file in (log_out, log_err)]
```

Comment:
`file()` was a builtin function until python 3 (https://docs.python.org/2/library/functions.html#file). Although it's no longer a builtin, my preference would be to avoid using it as a variable name.

Same for uses of `file` further down.

Reply:
@jnewbery Will open a new PR shortly and rename from `file` to `log_file`, `test_file`, etc. where applicable.